source/_docs/ecosystem/backup/backup_dropbox.markdown

```diff
@@ -37,6 +37,16 @@ Follow the instructions you see on your screen.
 
 Go to the folder you have placed `dropbox.py`.
 
+Option A:
+  Copy file `dropbox_uploader.sh` to : `.homeassistant/extraconfig/shell_code/` (so the full path would be similar to: `/home/homeassistant/.homeassistant/extraconfig/shell_code/dropbox_uploader.sh`)
+Option B:
+  Edit `dropbox.py`:
+  Change the following line: 
+  `uploader = "/home/homeassistant/.homeassistant/extraconfig/shell_code/dropbox_uploader.sh"`
+  to where you placed your file: (for example):
+  `uploader = "/home/homeassistant/MyFolder/dropbox_uploader.sh"`
+
```

Comment:
This looks terrible when [previewed](https://deploy-preview-5885--home-assistant-docs.netlify.com/docs/ecosystem/backup/backup_dropbox/). Please adjust the formatting

Reply:
Any better?